generated/thrift/rpc.thrift

```diff
@@ -104,8 +111,13 @@ struct HealthResult {
 	2: required string status
 }
 
+struct IDWithNamespace {
+    1: required string ns
+    2: required string id
+}
+
 struct WriteRequest {
-	1: required string id
+    1: required IDWithNamespace idWithNamespace
```

Comment:
Any way we can avoid sending the full namespace string along with each write?  This might up our total traffic out of the machine by up to 20% or so considering the ID string makes up the majority of a write's payload and the namespace string could be as long as 20% of the full ID.